README.md

```diff
@@ -18,7 +18,7 @@ Development builds are also available from [our website](http://pcsx2.net/downlo
 ## Minimum
 * OS: Windows Vista SP2 or newer or GNU/Linux (32-bit or 64-bit)
 * CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up)
-* GPU: DirectX 10 GPU or better
+* GPU: DirectX 10 GPU or better (on linux Required Opengl 3.3)
```

Comment:
Its "Linux" with an uppercase "L", lowercase "r" in "required" and "OpenGL".

You might just rephrase it as "(or OpenGL 3.3 or better on Linux)"

Reply:
"I'd just like to interject for a moment...."

Reply:
Sorry, I didn't follow the conversation. My bad.